tests/renderRepoCard.test.js

```diff
@@ -255,4 +255,20 @@ describe("Test renderRepoCard", () => {
     expect(queryByTestId(document.body, "stargazers")).toBeNull();
     expect(queryByTestId(document.body, "forkcount")).toBeNull();
   });
+  
+  it("should render template", () => {
+    document.body.innerHTML = renderRepoCard({
+      ...data_repo.repository,
+      isTemplate: true
+    });
+    expect(queryByTestId(document.body, "template-badge")).toBeDefined();
```

Comment:
I have no idea why this test is passing, because there is no template-badge data-testid

Reply:
Ahhh got it, i think .toBeDefined is not the right assertion to do, it should be toBeInTheDocument, and i think that means all other toBeDefined also should be changed. because null is a possible value & toBeDefined only fails if the value is "undefined"